layerize.js

```diff
@@ -577,14 +577,13 @@ function processFile(fileName, data) {
             chars.push({unicode: unicodes[0], components: layers});
         } else {
             // Ligatures: Keycap sequences only apply when there's U+FE0F in between.
-            // Otherwise, if not a Regional-Indicator pair or Fitzpatrick-modified char,
-            // insert ZWJ between components
+            // Twemoji already contains ZWJ characters
             if (unicodes.length == 2 && parseInt(unicodes[1], 16) == 0x20e3) {
                 unicodes = unicodes.join(",fe0f,").split(",");
```

Comment:
Can you check if the keycap sequences come with 0xfe0f in their file names already? If so we could remove this `if` block.

Reply:
It seems that they do.